common/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Mail_Domain.php

```diff
@@ -15,3 +15,4 @@
 $L['OpenDkimStatus_label'] = 'Sign outbound messages with DomainKeys Identified Mail (DKIM)';
 $L['DkimSelector_label'] = '1. Add a TXT record to your public DNS service provider with key "${0}"';
 $L['DkimKey_label'] = '2. Copy and paste the following text in the record data (RDATA) section';
+$L['Cannot_enable_dkim_and_disclaimer_both'] = 'The DISCLAIMER and the DKIM settings are not compatible';
```

Comment:
I'd change to something like: The legal note and DKIM settings can't be both enabled at the same time.

Reply:
I agree, it's better to reflect the same wording. In this context the "disclaimer" is labeled as "legal note".